File: Sample/ioPatternLockSample/Screens/Main/VireControllers/MainViewController.m
//
//  MainViewController.m
//  ioPatternLockSample
//
//  Created by Ilker OZCAN on 27.11.2018.
//  Copyright © 2018 Ilker OZCAN. All rights reserved.
//

#import <IOPatternLock/IOPatternLock.h>
#import "MainViewController.h"

@interface MainViewController () <IOPatternLockDelegate>

@property (nonatomic, weak) IBOutlet IOPatternLockView *patternLockView;

@end

@implementation MainViewController

#pragma mark - Pattern Lock Delegate

- (void)IOPatternLockView:(IOPatternLockView *)patternLockView patternCompleted:(NSArray<NSNumber *> *)selectedPatterns {
	NSLog(@"Pattern completed.\nSelected patterns %@", selectedPatterns);
}

- (void)IOPatternLockView:(IOPatternLockView *)patternLockView patternCompletedWithError:(NSError *)error {
	NSLog(@"Pattern error.\n%@", error.localizedDescription);
	[patternLockView redraw];
}

@end

File: SwiftSample/IOPatternLockSwiftSample/Screens/Main/VireControllers/MainViewController.swift
//
//  MainViewController.swift
//  IOPatternLockSwiftSample
//
//  Created by Ilker OZCAN on 1.12.2018.
//  Copyright © 2018 Ilker OZCAN. All rights reserved.
//

import Foundation
import IOPatternLock
import UIKit

class MainViewController: UIViewController, IOPatternLockDelegate {
	
	@IBOutlet weak var patternLockView: IOPatternLockView!
	
	func ioPatternLockView(_ patternLockView: IOPatternLockView, patternCompleted selectedPatterns: [NSNumber]) {
		print(selectedPatterns)
	}
	
	func ioPatternLockView(_ patternLockView: IOPatternLockView, patternCompletedWithError error: Error) {
		print(error.localizedDescription)
	}
	
}
